roles/ceph-common/tasks/logging.yml

```diff
@@ -4,8 +4,16 @@
   args:
     options:
       - daily
-      - missingok
       - rotate 7
+      - daily
       - compress
+      - sharedscripts
+      - postrotate
+      -     killall -q -1 ceph-mon ceph-mds ceph-osd ceph-fuse radosgw || true
```

Comment:
ceph logrotate conf is comming from community, https://github.com/ceph/ceph/blob/master/src/logrotate.conf